### PR TITLE
[trivial] Add missing copyright headers

### PR DESCRIPTION
**Pull request by maflcko**

### CODE REVIEW

File: src/chainparamsseeds.h [hunk 1]
```diff
@@ -1,3 +1,7 @@
+// Copyright (c) 2014-2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
```
Comment (luke-jr) [inline src/chainparamsseeds.h]: This file is not eligible for copyright IMO.

File: qa/rpc-tests/test_framework/coverage.py [hunk 2]
```diff
@@ -1,3 +1,9 @@
+#!/usr/bin/env python2
+# Copyright (c) 2015 The Bitcoin Core developers
+# Distributed under the MIT/X11 software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+
```
Comment (luke-jr) [inline qa/rpc-tests/test_framework/coverage.py]: @jamesob This requires your ACK following #6804

File: src/consensus/merkle.cpp [hunk 3]
```diff
@@ -1,3 +1,7 @@
+// Copyright (c) 2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
```
Comment (luke-jr) [inline src/consensus/merkle.cpp]: Requires ACK from @sipa

File: src/httprpc.cpp [hunk 4]
```diff
@@ -1,3 +1,7 @@
+// Copyright (c) 2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
```
Comment (luke-jr) [inline src/httprpc.cpp]: Requires ACK from @laanwj and @instagibbs (see #7044)

File: src/prevector.h [hunk 5]
```diff
@@ -1,3 +1,7 @@
+// Copyright (c) 2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
```
Comment (luke-jr) [inline src/prevector.h]: Requires ACK from @sipa

File: src/httprpc.cpp (hunk 4 above)
Comment (instagibbs) [inline src/httprpc.cpp]: ACK

File: src/test/test_bitcoin.h [hunk 6]
```diff
@@ -1,3 +1,7 @@
+// Copyright (c) 2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
```
Comment (luke-jr) [inline src/test/test_bitcoin.h]: Requires ACK from @laanwj @gavinandresen and @morcos

File: src/torcontrol.cpp [hunk 7]
```diff
@@ -1,3 +1,7 @@
+// Copyright (c) 2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
```
Comment (luke-jr) [inline src/torcontrol.cpp]: Requires ACK from @laanwj @petertodd

Comment (petertodd) [inline src/torcontrol.cpp]: ACK src/torcontrol.cpp

**Comment by luke-jr** [comment]
Copyright holders please ACK each file being licensed explicitly. (Of course, you can also refuse, but that will disrupt 0.12 as we'd need to remove your code, so please don't...)

**Comment by morcos** [comment]
permanent ACK on all code I contribute to this repository having the preferred copyright of the project.

File: qa/rpc-tests/test_framework/coverage.py (hunk 2 above)
Comment (jamesob) [inline qa/rpc-tests/test_framework/coverage.py]: ACK

**Comment by gavinandresen** [comment]
permanent ACK on all code I have contributed or will contribute in the future for whatever copyright changes are deemed necessary or desirable by whoever is willing to spend precious brainpower worrying about copyright stuff.

**Comment by sdaftuar** [comment]
permanent ACK on all code I contribute to this repository having the preferred copyright of the project

**Comment by MarcoFalke** [comment]
@luke-jr You don't have to explicitly express your wish to have your contribution to Bitcoin Core attributed to you and licensed under the MIT license. By creating a pull request you implicitly agree to the statement in the README.md: https://github.com/bitcoin/bitcoin#license

The copyright header in each file serves the purpose to redundantly and explicitly list the contributors and the license for the case a file gets distributed on its own. Am I wrong?

**Comment by luke-jr** [comment]
So who wants to go through and make sure all this code was PR'd by the authors themselves?

If we want an implicit agreement, we should probably make a stronger statement to that effect somewhere...

**Comment by luke-jr** [comment]
(note that in the past, certain people have in fact PR'd things which were written by another person and was explicitly *not* MIT-licensed...)

**Comment by laanwj** [comment]
ACK everything I've contributed to Bitcoin Core as MIT, apart from some images (as described in assets-attibution)

**Comment by instagibbs** [comment]
permanent ACK on all code I contribute to this repository having the preferred copyright of the project

**Comment by jamesob** [comment]
permanent ACK on all code I contribute to this repository having the preferred copyright of the project

**Pull request merged**